src/main/java/teammates/ui/controller/InstructorFeedbackEditCopyData.java

```diff
@@ -0,0 +1,30 @@
+package teammates.ui.controller;
+
+import teammates.common.datatransfer.AccountAttributes;
+
+/**
+ * PageData object that is sent back as a result of InstructorFeedbackEditCopy. 
+ * Specifies a page for the user to be redirected to as a result of the form submission. 
+ *
+ */
+public class InstructorFeedbackEditCopyData extends PageData {
+    public final String redirectUrl;
+    public final String errorMessage;
+    public final boolean isError;
+    
+    
+    public InstructorFeedbackEditCopyData(AccountAttributes account, 
+                                          String redirectUrl, String errorMessage) {
+        super(account);
+        this.redirectUrl = redirectUrl != null ? redirectUrl : "";
+        this.errorMessage = errorMessage;
+        this.isError = !errorMessage.isEmpty();
```

Comment:
Is `isError` needed?
Though it does give a confirmation that there is an error, it seems fine just checking if `errorMessage` is empty or not from wherever this class is being used from instead.